subsys/bluetooth/controller/Kconfig

```diff
@@ -497,6 +497,22 @@ config BT_CTLR_SYNC_PERIODIC
 config BT_CTLR_SYNC_PERIODIC
 	bool "LE Periodic Advertising in Synchronization State [EXPERIMENTAL]" if BT_LL_SW_SPLIT
 
+config BT_CTLR_SYNC_PERIODIC_ADV_LIST
+	bool "LE Periodic Advertiser List support"
+	depends on BT_CTLR_SYNC_PERIODIC
+	default y
+	help
+	  Enable support for LE Periodic Advertiser List support.
+
+config BT_CTLR_SYNC_PERIODIC_ADV_LIST_SIZE
```

Comment:
Isn't this config a redundant to the "CONFIG_BT_PER_ADV_SYNC_MAX"?

Reply:
No, maximum supported simultaneous synchronizations can be lower than listed number of advertisers; only one advertiser is synchronized for every LE Periodic Advertising Create Sync Command, and _Memory Capacity Exceeded_ error is used to convey insufficient resources to handle any more periodic advertising trains.

Reply:
Got it! so setting the "CONFIG_BT_PER_ADV_SYNC_MAX" to a number greater than the "BT_CTLR_SYNC_PERIODIC_ADV_LIST_SIZE" is useless, right?

Reply:
Maximum simultaneous synchronizations `CONFIG_BT_PER_ADV_SYNC_MAX` can be greater than, says number of trusted peer device stored in a list `CONFIG_BT_CTLR_SYNC_PERIODIC_ADV_LIST_SIZE`. List is just a convenience, to be able to synchronize to any of the listed peers on a first-seen basis. Applications are permitted to use explicit peer device address to create additional synchronizations beyond the number of entries that be accommodated in the Periodic Advertiser List.